src/Symfony/Component/RateLimiter/Limit.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\RateLimiter;
+
+/**
+ * @author Valentin Silvestre <vsilvestre.pro@gmail.com>
+ *
+ * @experimental in 5.2
+ */
+final class Limit
```

Comment:
Because it would kill any extensibility, a VO cannot be final when it's part of an interface (as it is in LimitInterface).

Reply:
Oops, I should use other templates for OSS :D

Reply:
these templates should be fixed :)